keps/sig-scheduling/5055-dra-device-taints-and-tolerations/README.md

```diff
@@ -535,8 +676,39 @@ type DeviceTaintSelector struct {
     // +listType=atomic
     Selectors []DeviceSelector
 }
+
+// DeviceTaintRuleStatus provides information about an on-going pod eviction.
+type DeviceTaintRuleStatus struct {
+    // Conditions provide information about the current state of the DeviceTaintRule
+    // in a machine-readable and human-readable format.
+    //
+    // The following condition is currently defined as part of this API, more may
+    // get added:
+    // - Type: EvictionInProgress
```

Comment:
I would add `Ready` or `Completed` condition state.

Reply:
Can you describe what the semantic of those states would be?

"Completed" could replace "EvictionInProgress", but it could a bit misleading: it might be "Completed" *now*, but in the unlikely case that some pod pops up which is scheduled (for example, the scheduler missed the taint or a user manually scheduled a pod), then it's possible to go back from "Completed: True" to "Completed: False". It's not really a final state.